File: AndroidSimple/src/main/java/com/pinger/repertory/recycler/RecyclerPageActivity.java
package com.pinger.repertory.recycler;

import android.view.View;

import com.pinger.baselib.base.BaseActivity;
import com.pinger.repertory.R;

/**
 * @author Pinger
 * @since 2018/1/13 0013 上午 11:09
 */

public class RecyclerPageActivity extends BaseActivity {
    @Override
    protected int getLayoutResID() {
        return R.layout.activity_recycler_page;
    }

    @Override
    public void onClick(View view) {
        if (view.getId() == R.id.btn_recycler_view) {
            startActivity(RecyclerViewActivity.class);
        }
    }

    @Override
    protected void initView() {
        setActionBar(getString(R.string.recycler_page), true);
    }
}

File: sample/src/main/java/com/fungo/repertory/recycler/RecyclerPageActivity.kt
package com.fungo.repertory.recycler

import com.fungo.baselib.base.page.BasePageActivity
import com.fungo.baselib.base.page.swipeback.SwipeBackLayout
import com.fungo.repertory.R

/**
 * @author Pinger
 * @since 2018/1/13 0013 上午 11:09
 */

class RecyclerPageActivity : BasePageActivity() {
    override val layoutResID: Int
        get() = R.layout.activity_recycler_page


    override fun initView() {
        setEdgeLevel(SwipeBackLayout.EdgeLevel.MED)
    }
}
